docs/_includes/chart.html

```diff
@@ -0,0 +1,602 @@
+                            role="graphics-symbol" tabindex="-1"
+                            aria-labelledby="ebay-chart-c6-chart-legend-label-1"></path>
+                        <path class="donut-graph__segment donut-graph__segment--secondary"
+                            d="M 19.77093832475971 58.308335013754245 A 83.7 83.7 0 0 1 83.35121144964256 12.114562646330555 Q 84.50960835066778 11.959999499978196 84.21820858263077 9.653332819422033 L 83.63540904655676 5.0399994583097225 Q 83.34400927851975 2.73333277775356 82.05690161071396 2.9050696070339512 A 93 93 0 0 0 11.412153694177462 54.23148334861583 Q 10.851084120660175 55.40252588444828 12.954807017643674 56.39246273733707 L 17.162252811610657 58.372336443114655 Q 19.265975708594155 59.36227329600345 19.77093832475971 58.308335013754245"
+                            role="graphics-symbol" tabindex="-1"
+                            aria-labelledby="ebay-chart-c6-chart-legend-label-2"></path>
+                    </svg></div>
+                <div class="chart-legend__container chart-legend__container-vertical">
+                    <div container-id="ebay-chart-c6" class="chart-legend">
+                        <dl>
+                            <div id="ebay-chart-c6-chart-legend-label-0" 
+                                class="chart-legend__item chart-legend__item--primary">
+                                <dt>Value 1</dt>
+                                <dd class="value">$ 3<span class="clipped">50.00%</span></dd>
+                            </div>
+                            <div id="ebay-chart-c6-chart-legend-label-1" 
+                                class="chart-legend__item chart-legend__item--tertiary">
+                                <dt>Value 2</dt>
+                                <dd class="value">$ 2<span class="clipped">33.33%</span></dd>
+                            </div>
+                            <div id="ebay-chart-c6-chart-legend-label-2" 
+                                class="chart-legend__item chart-legend__item--secondary">
+                                <dt>Value 3</dt>
+                                <dd class="value">$ 1<span class="clipped">16.67%</span></dd>
+                            </div>
+                        </dl>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+
+    {% highlight html %}
+    <div aria-label="Donut chart title" class="chart chart--composition chart--stacked">
```

Comment:
Looks like the example code is out of sync with the real code above. E.g. `.chart--composition` class.

Reply:
good catch, missed this during some cleanup